src/Toolkit/Toolkit.UI.Controls/ScaleLine/ScaleLine.Windows.cs

```diff
@@ -55,7 +60,12 @@ public override void OnApplyTemplate()
             _metricScaleLine = GetTemplateChild("MetricScaleLine") as Rectangle;
             Refresh();
         }
-
+#if WPF
+        protected override AutomationPeer OnCreateAutomationPeer()
+        {
+            return new ScaleLineAutomationPeer(this);
```

Comment:
I think you're missing a call in ScaleLine to announce the value changes on the automation peer when the scale changes. Something like
`this?._automationPeer.RaisePropertyChangedEvent(ValuePatternIdentifiers.ValueProperty, oldValue, newValue)`
This would then also mean you'd need to cache the created automation peer here for later event notification. ie something like `return _automationPeer ?? _automationPeer = new ScaleLineAutomationPeer(this);`